cards/src/UserCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import getUsers from './getUsers';
+
+const useStyles = makeStyles({
+  root: {
+    maxHeight: 1000,
+    maxWidth: 545,
+  },
+});
+
+const UserCard = () => {
+  const [user, setUser] = React.useState({
+    avatar:'loading...',
```

Comment:
When the page first loads, the image appears as a broken link, because `'loading...'` isn't a valid src. Given that you have the `isLoading` state below, I think I'd be more inclined to set all of these default values to `null`, and then conditionally render a loading message if the component `isLoading`

```jsx
if(isLoading){
  return <p>Loading...</p>;
}

// rest of the render
```